src/modules/featurepopup/templates/popup.html

```diff
@@ -1,6 +1,18 @@
-<div class="anol-popup {{ openingDirection }}" ng-mousedown="makeDraggable($event)">
-    <span class="anol-popup-closer glyphicon glyphicon-remove" ng-click="close()" ng-if="!sticky" ng-mousedown="$event.stopPropagation();"></span>
-    <div class="anol-popup-content">
-        <ng-transclude></ng-transclude>
+<div ng-show="isVisible">
+    <div class="anol-popup {{ openingDirection }} {{ class }}"
```

Comment:
it might be better to use `ng-class` here to make sure the interpolated values will be set correctly

Reply:
I don't really know how to do that here, because there already is `ng-class` here that uses the object syntax.

Reply:
also class will always be a string of class names that is separated by spaces. openingDirection is always a string